posting/list.go

```diff
@@ -77,11 +78,333 @@ type List struct {
 	x.SafeMutex
 	key         []byte
 	plist       *pb.PostingList
-	mutationMap map[uint64]*pb.PostingList
+	mutationMap *MutableLayer
 	minTs       uint64 // commit timestamp of immutable layer, reject reads before this ts.
 	maxTs       uint64 // max commit timestamp seen for this list.
 }
 
+// MutableLayer is the structure that will store mutable layer of the posting list. Every posting list has an immutable
```

Comment:
nit: can connect it better to the fields of the structs in `List` and `MutableLayer`